.github/workflows/github-actions.yml

```
@@ -0,0 +1,99 @@
+# ✅ GitHub Actions: 변경된 서비스만 ECR로 Push + EC2에서 Pull & Run (with .env)
+name: CI/CD with GitHub Actions, Docker, and ECR
+
+on:
+  push:
+    branches: [ "feature/profile-setting" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)